gnucash/import-export/import-main-matcher.c

```diff
@@ -977,6 +979,15 @@ GNCImportMainMatcher *gnc_gen_trans_list_new (GtkWidget *parent,
     gtk_builder_connect_signals_full (builder, gnc_builder_connect_full_func, info);
 
     g_object_unref (G_OBJECT(builder));
+    
+    // Register this UI, it needs to be closed when the session is closed.
+    id = gnc_register_gui_component(IMPORT_MAIN_MATCHER_CM_CLASS,
+                                    NULL, /* no refresh handler */
+                                    (GNCComponentCloseHandler)gnc_gen_trans_list_delete,
+                                    info);
+    // This ensure this dialog is closed when the session is closed.
+    gnc_gui_component_set_session (id,gnc_get_current_session());
+
```

Comment:
looks like an extra blank line crept in and can you add a space after gnc_register_gui_component and adjust other line accordingly.